metagraph/src/graph/representation/succinct/dbg_succinct.hpp

```diff
@@ -14,6 +16,15 @@ class DBGSuccinct : public DeBruijnGraph {
   public:
     friend class MaskedDeBruijnGraph;
 
+    /**
+     * A wrapper which caches computed node sequences for DBGSuccinct graphs.
+     * This allows for faster get_node_sequence and call_incoming_kmers calls.
+     * Traversal operations and map_to_nodes_sequentially are overridden to fill
+     * the cache alongside their normal functions.
+     * This wrapper stores ~144 bytes per cached node.
```

Comment:
doesn't this depend on the k-mer length?

Reply:
It does, so it'll be either 144, 152, or 168 depending in which kmertype is used

Reply:
I updated this now to reflect the new changes.